scripts/utils/utils.tcl

```diff
@@ -486,6 +486,10 @@ proc run_tcl_script {args} {
     set index 1
     set name $::env(DESIGN_NAME)
 
+    if { [info exists flag_map(-def_in)] } {
```

Comment:
Why not make IO_READ_DEF a flag and then just switch to CURRENT_DEF? Wouldn't that be cleaner?

Reply:
I made some changes. Are they as you would expect ?